### PR TITLE
chore(a380): add a380 fire protection ECAM messages ATA26

### PR DESCRIPTION
**Pull request by mjuhe**
Contributing to #8739
Based on #8750 

## Summary of Changes

Adds abnormal checklists, inop systems, memos and Pseudo FWC logic for the fire protection system. This PR is based on #8750 so it needs to be merged into that or rebased on master.

## Additional context

Discord username (if different from GitHub): Commander Ga

## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page

### CODE REVIEW

**Comment by flogross89** [comment]
setting this to "do not merge", as long as it's targeted against master and the ECAM PR isn't merged yet :)

**Review by 2hwk — APPROVED** [review]

**Comment by mjuhe** [comment]
Sorry this should've been in draft as I'm still working on adding the PseudoFWC logic to it and the remaining procedures.

File: fbw-a380x/src/systems/instruments/src/MsfsAvionicsCommon/EcamMessages/AbnormalSensed/ata26.tsx [hunk 1]
```diff
@@ -16,4 +16,2039 @@ import { AbnormalProcedure } from 'instruments/src/MsfsAvionicsCommon/EcamMessag
...
+        labelNotCompleted: 'DISCH',
+      },
+      {
+        name: 'ATC',
+        sensed: false,
+        labelNotCompleted: 'NOTIFY',
+      },
+      {
+        // When timer elapsed, this should change to 'AS STILL FIRE AFTER 30 S'
+        name: '.IF STILL FIRE AFTER 30 S :',
```
Comment (flogross89) [inline fbw-a380x/src/systems/instruments/src/MsfsAvionicsCommon/EcamMessages/AbnormalSensed/ata26.tsx:101]: We don't need a dot in front of the condition on the A380. Also valid for all the other conditions, should be fixed with a search & replace on ".IF"

File: fbw-a380x/src/systems/systems-host/systems/FlightWarningSystem/FwsCore.ts [hunk 2]
```diff
@@ -2593,24 +2687,31 @@ export class FwsCore implements Instrument {
...
+          this.eng4FireDetected.get() ||
+          this.mlgFireDetected.get() ||
           this.emergencyGeneratorOn.get() ||
           (this.engine1State.get() === 0 && this.engine2State.get() === 0) ||
           (this.greenLP.get() && this.yellowLP.get()) ||
           (this.yellowLP.get() && this.blueLP.get()) ||
           (this.greenLP.get() && this.blueLP.get())),
     );
 
+    // Is this needed?
```
Comment (flogross89) [inline fbw-a380x/src/systems/systems-host/systems/FlightWarningSystem/FwsCore.ts:2703]: Good question, don't know why it was needed on the A32NX in the first place. I'd propose we keep it there and if it causes any trouble we can remove it

**Review by flogross89 — COMMENTED** [review]
Amazing work! Just one small thing left for the conditions

File: fbw-a380x/src/systems/instruments/src/MsfsAvionicsCommon/EcamMessages/AbnormalSensed/ata26.tsx (hunk 1 above)
Comment (mjuhe) [reply fbw-a380x/src/systems/instruments/src/MsfsAvionicsCommon/EcamMessages/AbnormalSensed/ata26.tsx]: Ah cool, makes it simpler, fixed now! Thanks.

File: fbw-a380x/src/systems/systems-host/systems/FlightWarningSystem/FwsCore.ts (hunk 2 above)
Comment (mjuhe) [reply fbw-a380x/src/systems/systems-host/systems/FlightWarningSystem/FwsCore.ts]: Sounds good and agreed, I've removed the comment.

**Review by flogross89 — APPROVED** [review]

**Pull request merged**